### PR TITLE
Bugfix when API key is not present at all

### PR DESCRIPTION
**Pull request by Gasta88**
This pull request addresses issue #1440 and fix the bug found in PR #1569

### CODE REVIEW

File: Tests/test_Entrez_online.py [hunk 1]
```diff
@@ -45,7 +45,6 @@
 URL_EMAIL = "email=biopython%40biopython.org"
 URL_API_KEY = "api_key=5cfd4026f9df285d6cfc723c662d74bcbe09"
 
-
```
Comment (peterjc) [inline Tests/test_Entrez_online.py]: Can you put this blank line back? This is why the TravisCI style checks failed:

```
flake8 Tests/
Tests/test_Entrez_online.py:48:1: E302 expected 2 blank lines, found 1
```

**Comment by peterjc** [comment]
Is this OK now @mdehoon?

**Comment by mdehoon** [comment]
Just wondering, why was the order of the imports changed in Tests/test_Entrez_online.py ? If there is no particular purpose, then I would suggest to keep the original order, so we don't inadvertently introduce a new bug.

**Comment by peterjc** [comment]
Reordering the imports here is a style choice - wider Python convention is to put standard library imports first, then 3rd party imports, then local imports.

It looks harmless here, but I'll retest with that locally...

**Comment by peterjc** [comment]
I've added a test - I've tried to avoid any side effects due to the global nature of the ``Entrez.api_key`` setting.

Over to @mdehoon to review & merge.

**Comment by mdehoon** [comment]
Merged, thanks!

**Pull request merged**